oscrypto/keys.py

```diff
@@ -21,4 +27,100 @@
     'parse_pkcs12',
     'parse_private',
     'parse_public',
+    "make_pkcs12",
 ]
+
+
+def make_pkcs12(key, cert, other_certs, password):
```

Comment:
Also, there should be a doc comment here describing what each parameter should contain, both the data type and an English description of the contents